package.json

```diff
@@ -43,84 +43,93 @@
+    "codesandbox/pacote": "^12.0.0",
+    "cssnano/**/css-select/nth-check": "^2.0.1",
+    "cssnano/**/postcss": "^7.0.39",
+    "enzyme/cheerio/cheerio-select-tmp/css-select/css-what": "^6.1.0",
+    "enzyme/cheerio/cheerio-select-tmp/css-what": "^6.1.0",
+    "html-webpack-plugin/**/ansi-regex": "^5.0.1",
+    "jest-cli/**/ansi-regex": "^5.0.1",
+    "jest/**/node-notifier": "^10.0.1",
+    "node-sass/sass-graph/scss-tokenizer": "^0.4.3",
+    "postcss-cli/chokidar/glob-parent": "^6.0.1",
+    "postcss-inline-svg/css-select/nth-check": "^2.0.1",
+    "react-view/**/ansi-regex": "^5.0.1",
+    "react-view/**/minimist": "^1.2.6",
+    "react-view/@miksu/prettier/minimatch": "^3.0.8",
+    "remark-parse/trim": "0.0.3",
+    "sass-lint-auto-fix/**/minimist": "^1.2.6",
+    "sass-lint-auto-fix/merge": "^2.1.1",
+    "sass-lint/**/minimist": "^1.2.6",
+    "sass-lint/eslint": "^7.10.0",
+    "sass-lint/front-matter": "^4.0.2",
+    "sass-lint/merge": "^2.1.1",
+    "start-server-and-test/**/minimist": "^1.2.6",
+    "webpack-dev-server/**/ansi-regex": "^5.0.1",
+    "webpack-dev-server/chokidar/glob-parent": "^6.0.1",
     "webpack-dev-server/selfsigned": "^2.0.1",
-    "**/eslint/shelljs": "^0.8.5",
-    "**/css-select/css-what": "^6.1.0",
-    "**/css-select/nth-check": "^2.0.1",
-    "**/cheerio-select-tmp/css-what": "^6.1.0",
-    "**/cheerio-select-tmp/nth-check": "^2.0.1",
-    "**/find-versions": "^4.0.0",
-    "codesandbox/**/pacote": "^12.0.0",
-    "yo/meow": "^9.0.0",
-    "**/stylelint/micromatch": "^3.1.0",
-    "webpack/**/serialize-javascript": "^3.1.0",
-    "**/scss-tokenizer": "^0.4.3",
-    "**/node-notifier": "^10.0.1",
-    "**/got": "^11.8.5",
-    "**/postcss": "^7.0.39",
-    "**/eslint": "^7.10.0",
-    "**/har-validator": "^5.1.5",
-    "**/d3-color": "^3.1.0",
-    "**/@miksu/prettier/minimatch": "^3.0.8",
-    "**/request/qs": "^6.11.0",
-    "**/make-fetch-happen": "^11.0.3",
-    "nodegit/ramda": "^0.27.1",
-    "sass-lint/front-matter": "^4.0.2"
+    "webpack/**/chokidar/glob-parent": "^6.0.1",
+    "webpack/terser-webpack-plugin/serialize-javascript": "^3.1.0",
+    "yo/**/find-versions": "^4.0.0",
+    "yo/**/got": "^11.8.5",
+    "yo/meow": "^9.0.0"
   },
   "pre-commit": [
     "test-staged"
   ],
   "dependencies": {
-    "@types/chroma-js": "^2.0.0",
-    "@types/lodash": "^4.14.160",
-    "@types/numeral": "^0.0.28",
-    "@types/react-beautiful-dnd": "^13.0.0",
-    "@types/react-input-autosize": "^2.2.0",
-    "@types/react-virtualized-auto-sizer": "^1.0.0",
-    "@types/react-window": "^1.8.2",
+    "@types/chroma-js": "^2.4.0",
+    "@types/lodash": "^4.14.191",
+    "@types/numeral": "^2.0.2",
+    "@types/react-beautiful-dnd": "^13.1.3",
+    "@types/react-input-autosize": "^2.2.1",
+    "@types/react-virtualized-auto-sizer": "^1.0.1",
+    "@types/react-window": "^1.8.5",
     "@types/refractor": "^3.0.0",
-    "@types/resize-observer-browser": "^0.1.5",
-    "@types/vfile-message": "^2.0.0",
```

Comment:
why has this been removed?

Reply:
> why has this been removed?

@types/vfile-message@2.0.0: This is a stub types definition. vfile-message provides its own type definitions